django_docker_engine/docker_utils.py

```diff
@@ -115,7 +123,17 @@ def kill(self, container):
                 ignore_errors=True
             )
 
+    def kill_lru(self):
+        '''
+        Sort the containers so that the least-recently-used ones are first,
+        and then kill them until the needed memory has been freed.
+        TODO: Merge the other branch so we know something about memory.
+        '''
+        self.list()
+        # TODO: Use Historian
```

Comment:
If you're going to get to these todos very soon, no worries. If not, I prefer issues especially if there's a chance someone else will work in this repo. On the refinery-platform, we decided to avoid To-Do for github issues.

Reply:
I should actually finish that one in this PR. Thanks!